assets/dev/scss/editor/_context-menu.scss

```diff
@@ -40,12 +44,17 @@
 			}
 
 			&__icon {
```

Comment:
Why you are now remove the code completely instead of to hide via CSS?

Reply:
This PR simply hides icons, it doesn't remove the functionality.